core/src/test/scala/integration/kafka/server/KRaftClusterTest.scala

```diff
@@ -96,6 +96,32 @@ class KRaftClusterTest {
     }
   }
 
+  @Test
+  def testCreateClusterAndRestartControllerNode(): Unit = {
```

Comment:
@soarez Yes, we shutdown the active controller, then restart it to make it send `ApiVersionRequest` to a standby controller.